packages/@coorpacademy-components/src/molecule/skill-picker-modal/index.js

```diff
@@ -0,0 +1,142 @@
+import React, {useMemo, useState, useEffect, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import BaseModal from '../base-modal';
+import Chip from '../../atom/chip';
+import Loader from '../../atom/loader';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const SkillPickerModal = (props, context) => {
+  const {
+    skills,
+    isOpen,
+    isLoading,
+    maxSelectedSkills = 6,
+    minSelectedSkills = 3,
+    onCancel,
+    onConfirm,
+    onClose
+  } = props;
+  const {translate} = context;
+
+  const [skillList, setSkillList] = useState([]);
+
+  const handleCancel = useCallback(() => {
+    setSkillList(JSON.parse(JSON.stringify(skills)));
+    onCancel();
+  }, [setSkillList, skills, onCancel]);
+
+  const handleClose = useCallback(() => {
+    setSkillList(JSON.parse(JSON.stringify(skills)));
+    onClose();
+  }, [setSkillList, skills, onClose]);
+
+  const selectedSkills = useMemo(() => skillList.filter(skill => skill.focus), [skillList]);
+  const isError = useMemo(
+    () => selectedSkills.length > maxSelectedSkills,
+    [selectedSkills, maxSelectedSkills]
+  );
+
+  const footer = useMemo(() => {
+    const footerDescription = isError
+      ? translate('skill_focus_footer_error_description').replace(
+          '.',
+          selectedSkills.length - maxSelectedSkills
+        )
+      : translate('skill_focus_footer_description').replace(
+          '.',
+          minSelectedSkills - selectedSkills.length
+        );
+    return {
+      text:
+        isLoading ||
+        (selectedSkills.length <= maxSelectedSkills && selectedSkills.length >= minSelectedSkills)
+          ? ''
+          : footerDescription.replace('.'),
+      isError,
+      cancelButton: {
+        onCancel: handleCancel,
+        label: translate('cancel'),
+        disabled: isLoading || isError
+      },
+      confirmButton: {
+        onConfirm: () => onConfirm(selectedSkills),
+        label: translate('confirm'),
+        iconName: 'circle-check',
+        disabled: isLoading || isError
+      }
+    };
+  }, [isError, handleCancel, onConfirm, translate, selectedSkills, isLoading, maxSelectedSkills]);
+
+  useEffect(() => {
+    if (skills && isOpen) {
+      setSkillList(JSON.parse(JSON.stringify(skills)));
+    }
+  }, [skills, isOpen, setSkillList]);
+
+  if (!isLoading && !skills && !isOpen) return null;
+
+  return (
+    <BaseModal
+      title={translate('skill_focus')}
+      description={translate('skill_focus_description')}
+      isOpen={isOpen}
+      onClose={handleClose}
+      footer={footer}
+      headerIcon={{
+        name: 'bullseye-arrow',
+        backgroundColor: '#DDD1FF'
+      }}
+    >
+      <div className={style.skillPickerContainer}>
+        {isLoading ? (
+          <div className={style.loaderContainer}>
+            <Loader className={style.loader} theme="coorpmanager" />
+          </div>
+        ) : (
+          <>
```

Comment:
Typo?

Reply:
I use Fragment so that I don't have to wrap them in an additional div